docs/lib/datastore/fragments/android/relational/save-snippet.md

```diff
@@ -52,5 +52,30 @@ Amplify.DataStore.save(post,
 )
 ```
 
+</amplify-block>
+<amplify-block name="RxJava">
+
+```java
+Post post = Post.builder()
+    .title("My Post with comments")
+    .rating(10)
+    .status(PostStatus.ACTIVE)
+    .build();
+
+Comment comment = Comment.builder()
+    .post(post) // Directly pass in the post instance
+    .content("Loving Amplify DataStore!")
+    .build();
+
+Completable.merge(
```

Comment:
:eyes: here please as I made this up.

Reply:
Looks legit to me.  Saving three things in parallel, waiting for all to finish.